src/libs/ReportUtils.js

```diff
@@ -980,6 +991,7 @@ function buildOptimisticIOUReport(ownerEmail, userEmail, total, chatReportID, cu
 
     return {
         // If we're sending money, hasOutstandingIOU should be false
+        type: CONST.REPORT.TYPE.IOU,
```

Comment:
Could you just move the comment to the relevant line? 

`hasOutstandingIOU: !isSendingMoney // If we're sending money, hasOutstandingIOU should be false`

Reply:
+1 I think the comment should be next as close to `hasOutstandingIOU: !isSendingMoney` as possible

Reply:
oh, my bad